.github/workflows/backend.yml

```diff
@@ -39,32 +39,8 @@ jobs:
             - '.github/**'
           scripts:
             - 'scripts/**'
-  client-lint:
-    needs: changes
-    if: needs.changes.outputs.python-client == 'true' ||  needs.changes.outputs.workflows == 'true'
```

Comment:
Why did we get rid of the "if"s from all of here and below? Won't that undo all the gains we get from #4515

Reply:
Yep you're right! Let me add back